Cog.entitlements

```diff
@@ -2,9 +2,17 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>com.apple.security.app-sandbox</key>
```

Comment:
Permissions requested:
1. Incoming Connections (http stream?)
2. User's Music directory read-only access (play user's music)
3. Drag&droped and opened files read-only access (play user's music)

I think this is sufficient for now.

Reply:
Will also need to store bookmarks for those dragged and dropped or opened music files, unless the XML playlist is already sufficient for that.